src/app/pages/AccountPage/Features/SendTransaction/__tests__/index.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import * as React from 'react'
+import { Provider } from 'react-redux'
+import { configureAppStore } from 'store/configureStore'
+import { SendTransaction } from '..'
+
+jest.mock('react-i18next', () => ({
+  useTranslation: () => {
+    return {
+      t: str => str,
+      i18n: {
+        changeLanguage: () => new Promise(() => {}),
+      },
+    }
+  },
+}))
+
+const renderComponent = store =>
+  render(
+    <Provider store={store}>
+      <SendTransaction />
+    </Provider>,
+  )
+
+describe('<SendTransaction />', () => {
```

Comment:
I think it would ease testing a bunch if it was implemented more like `expectConfirmation`, but we can figure that out later